source/appModules/soffice.py

```diff
@@ -431,10 +447,16 @@ class SymphonyDocument(CompoundDocument):
 	TextInfo = SymphonyDocumentTextInfo
 
 	# variables used for handling announcements resulting from gestures
-	GESTURE_ANNOUNCEMENT_TIMEOUT = 0.15
-	announceToolbarButtonToggle = False
```

Comment:
this appears to be an API breaking change. Can you please document it in the API breaking changes section of the change log.

Reply:
Done now. (I wasn't really aware that app modules are part of the public API as well.)

Reply:
thanks - they are, its rarely an issue but better be safe than sorry 😄